docs/book/v4/tutorials/create-book-module.md

```diff
@@ -0,0 +1,608 @@
+                      \Api\Book\RoutesDelegator::class,
+                  ],
+              ],
+              'factories'  => [
+                  ...
+              ]
+              ...
+
+* In `src/config/autoload/doctrine.global.php` add this under the `doctrine.driver` key:
+
+      'BookEntities'  => [
+          'class' => AttributeDriver::class,
+          'cache' => 'array',
+          'paths' => __DIR__ . '/../../src/Book/src/Entity',
+      ],
+
+* add this under the `doctrine.driver` key:
+
+  ``'Api\\Book\Entity'    => 'BookEntities',`` add this under the `doctrine.driver.drivers` class
+
+Example:
+
+    <?php
+    ...
+    return [
+        'doctrine'            => [
+            ...
+            'driver'        => [
+                'orm_default'   => [
+                    'class'   => MappingDriverChain::class,
+                    'drivers' => [
+                        'Api\\App\Entity'    => 'AppEntities',
+                        'Api\\Admin\\Entity' => 'AdminEntities',
+                        'Api\\User\\Entity'  => 'UserEntities',
+                        'Api\\Book\Entity'    => 'BookEntities',
+                    ],
+                ],
+                'AdminEntities' => [
+                    'class' => AttributeDriver::class,
+                    'cache' => 'array',
+                    'paths' => __DIR__ . '/../../src/Admin/src/Entity',
+                ],
+                'UserEntities'  => [
+                    'class' => AttributeDriver::class,
+                    'cache' => 'array',
+                    'paths' => __DIR__ . '/../../src/User/src/Entity',
+                ],
+                'AppEntities'   => [
+                    'class' => AttributeDriver::class,
+                    'cache' => 'array',
+                    'paths' => __DIR__ . '/../../src/App/src/Entity',
+                ],
+                'BookEntities'  => [
+                    'class' => AttributeDriver::class,
+                    'cache' => 'array',
+                    'paths' => __DIR__ . '/../../src/Book/src/Entity',
+                ],
+            ],
+            ...
+
+Next we need to configure access to the newly created endpoints, add `books.list` and `book.create` to the authorization rbac array, under the `UserRole::ROLE_GUEST` key.
+> Make sure you read and understand the rbac documentation.
+
+## Migrations
+
+We created the `Book` entity, but we didn't create the associated table for it.
+
+Doctrine can handle the table creation, run the following command:
+
+    vendor/bin/doctrine-migrations diff --filter-expression='/^(?!oauth_)/'
+
+This will check for differences between your entities and database structure and create migration files if necessary, in `data/doctrine/migrations`.
+
+To execute the migrations run:
+
+    vendor/bin/doctrine-migrations migrate
+
+## Checking endpoints
+
+If we did everything as planned we can call the `http://localhost/books` endpoint and create a new book:
```

Comment:
If in the installation doc we instruct the developer to ue `http://0.0.0.0:8080` to access the new project, we should use the same pattern here as well.